torba/torba/client/basedatabase.py

```diff
@@ -461,15 +461,17 @@ def _transaction(conn: sqlite3.Connection, tx: BaseTransaction, address, txhash,
             " JOIN tx USING (txid)".format(cols), **constraints
         ))
 
-    async def get_txos(self, my_account=None, **constraints):
-        my_account = my_account or constraints.get('account', None)
-        if isinstance(my_account, BaseAccount):
-            my_account = my_account.public_key.address
+    async def get_txos(self, my_accounts, filtering_accounts=None, **constraints):
+        my_account_address = []
+        if my_accounts is not None:
+            my_account_address = [my_account.public_key.address for my_account in my_accounts if
+                                  isinstance(my_account, BaseAccount)]
+
         if 'order_by' not in constraints:
             constraints['order_by'] = ["tx.height=0 DESC", "tx.height DESC", "tx.position DESC"]
         rows = await self.select_txos(
             "tx.txid, raw, tx.height, tx.position, tx.is_verified, txo.position, chain, account",
-            **constraints
+            account=filtering_accounts, **constraints
```

Comment:
account is singular, filtering_accounts is plural, this seems like it will be a problem?

Reply:
SQL only accepts account as a filter currently, so instead of changing the SQL I used this approach

Reply:
Reason same as above.

Reply:
Then other stuff needs to be fixed or refactored. It just doesn't make sense that you have a variable that's clearly one item `account` and you are setting it to a list of accounts `filtering_accounts`.